data/json/recipes/weapon/magazines.json

```diff
@@ -24,8 +25,11 @@
     "time": "20 m",
     "book_learn": [ [ "recipe_bullets", 2 ], [ "mag_rifle", 3 ] ],
     "qualities": [ { "id": "SAW_M", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
-    "tools": [ [ [ "small_repairkit", 10 ], [ "large_repairkit", 5 ] ] ],
-    "components": [ [ [ "steel_chunk", 2 ] ] ]
+    "tools": [
+      [ [ "762_casing", -1 ], [ "762_m43", -1 ], [ "762_m87", -1 ], [ "reloaded_762_m43", -1 ], [ "reloaded_762_m87", -1 ] ],
```

Comment:
These recipes should all use casehead requirements to keep all the data together in one place.

Reply:
Done.